client/state/selectors/is-requesting-account-recovery-reset-options.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Return a boolean value indicating whether requesting for the password reset options is in progress.
+ *
+ * @param  {Object} state  Global state tree
+ * @return {Boolean} If the request is in progress
+ */
+
+export default ( state ) => {
+	return state.accountRecovery.reset.options.isRequesting;
```

Comment:
Might be useful to use lodash `get` here to provide a default when something is undefined here.